client/state/mobile-apps/actions.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { MOBILE_APPS_LOGIN_EMAIL_SEND } from 'state/action-types';
+import { getLanguage, getLocaleSlug } from 'lib/i18n-utils';
+import 'state/data-layer/wpcom/auth/send-login-email';
+
+export const sendMobileEmailLogin = ( email, { redirectTo, showGlobalNotices = false } ) => {
+	//Kind of weird usage, but this is a straight port from undocumented.js for now.
+	//I can move this to the caller, if there's equivalent info in the state tree
```

Comment:
I think @jsnajdr had some thoughts on where the authoritative l10n info should come from?

Reply:
Happy to update, if y'all had recommendations 👍